### PR TITLE
FHIRflat transform CLI for redcap -> fhirflat

### LINKED ISSUE #31: Create fhirflat-transform CLI script
CLI script to wrap the `fhirflat.ingest` module

### PR DESCRIPTION
**Pull request by pipliggins**
Fixes #31

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,7 +80,7 @@ from fhir.resources.patient import Patient
 patients = Patient.fhir_bulk_import("patient_export.ndjson")
 ```
 
-### To FHIRflat
+#### To FHIRflat
```
Comment (abhidg) [inline README.md:83]: I have moved these to docs

**Review by abhidg — APPROVED** [review]
Looks good! I have moved some of this info to docs - can fix those up later.

**Comment by pipliggins** [comment]
> ## [Codecov](https://app.codecov.io/gh/globaldothealth/fhirflat/pull/39?dropdown=coverage&src=pr&el=h1&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth) Report
> Attention: Patch coverage is `23.07692%` with `10 lines` in your changes are missing coverage. Please review.
> 
> > Project coverage is 91.69%. Comparing base [(`ef763fb`)](https://app.codecov.io/gh/globaldothealth/fhirflat/commit/ef763fb70c372c5469f3bcbf012d544da155dda3?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth) to head [(`0b9bd9f`)](https://app.codecov.io/gh/globaldothealth/fhirflat/commit/0b9bd9f44cb0edea768cc9fdf216ac01ebfe78dc?dropdown=coverage&el=desc&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth).
> > Report is 1 commits behind head on main.
> 
> [Files](https://app.codecov.io/gh/globaldothealth/fhirflat/pull/39?dropdown=coverage&src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth)	Patch %	Lines
> [fhirflat/ingest.py](https://app.codecov.io/gh/globaldothealth/fhirflat/pull/39?src=pr&el=tree&filepath=fhirflat%2Fingest.py&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth#diff-ZmhpcmZsYXQvaW5nZXN0LnB5)	23.07%	[10 Missing ⚠️ ](https://app.codecov.io/gh/globaldothealth/fhirflat/pull/39?src=pr&el=tree&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth)
> Additional details and impacted files
> [☔ View full report in Codecov by Sentry](https://app.codecov.io/gh/globaldothealth/fhirflat/pull/39?dropdown=coverage&src=pr&el=continue&utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth). 📢 Have feedback on the report? [Share it here](https://about.codecov.io/codecov-pr-comment-feedback/?utm_medium=referral&utm_source=github&utm_content=comment&utm_campaign=pr+comments&utm_term=globaldothealth).

Going to ignore this until we have some dummy data matching the format of the Dengue redcap data that we're cleared to upload to github so we can test alongside the dengue google sheet

File: README.md (hunk 1 above)
Comment (pipliggins) [reply README.md]: I've kept your changes and moved the CLI / raw data to FHIR stuff into a new page in the docs

**Review by abhidg — APPROVED** [review]
🎉

**Pull request merged**